repos/system_upgrade/common/actors/persistentnetnamesdisable/actor.py

```diff
@@ -50,6 +50,11 @@ def process(self):
                     title='How to perform an in-place upgrade to RHEL 8 when using kernel NIC names on RHEL 7',
                     url='https://access.redhat.com/solutions/4067471'
                 ),
+                reporting.ExternalLink(
+                    title='RHEL 8 to RHEL 9: inplace upgrade fails at '
```

Comment:
[nitpicking] usually you put `url` first

Reply:
it's ok, the order is not important in this case. let's keep it